### DISCUSSION TITLE
Spatie role and permission selected role on edit page.

### DISCUSSION THREAD

**Discussion by shankhadevpadam** [filamentphp/filament#7896] [General]
I have configured spatie role and permission in filament. But my issue is how to get a selected role on the edit page.

![Screenshot](https://github.com/filamentphp/filament/assets/3838722/a6ec88fd-f172-4a67-8fe7-bbc1c141add8)

**Comment by leandrocfe** [comment]
```php
Select::make('roles')
    ->multiple()
    ->relationship('roles', 'name')
```
?
